scratch-blocks/core/field_numberdropdown.js

```
@@ -74,4 +74,4 @@ Blockly.FieldNumberDropdown.fromJson = function(element) {
   );
 };
 
-Blockly.Field.register('field_numberdropdown', Blockly.FieldNumberDropdown);
+Blockly.Field.register('field_numberdropdown', Blockly.FieldNumberDropdown);
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)